lidarr/tag.go

```diff
@@ -92,7 +92,8 @@ func (l *Lidarr) DeleteTag(tagID int) error {
 
 func (l *Lidarr) DeleteTagContext(ctx context.Context, tagID int) error {
 	uri := path.Join(bpTag, strconv.Itoa(tagID))
-	if err := l.DeleteAny(ctx, uri, nil); err != nil {
+
+	if err := l.DeleteAny(ctx, &starr.Request{URI: uri}); err != nil {
```

Comment:
also here

Reply:
good call. I left a few of these so you could see the difference and tell me which you liked. Got it! :)